plugins/modules/user_role_workflow_manager.py

```diff
@@ -1274,7 +1276,7 @@ def get_diff_merged(self, config):
                 (consolidated_data, update_required_param) = self.user_requires_update(self.have["current_user_config"], self.have["current_role_id_config"])
 
                 if not consolidated_data:
-                    self.msg = "User does not need any update"
+                    self.msg = "User with username '{0}' already exist and the user does not need any update".format(self.have.get("username"))
```

Comment:
The word "exist" should be "exists" ..
`self.msg = "User with username '{0}' already exists and does not require an update.".format(self.have.get("username"))
`

Reply:
Updated the code accordingly.